.github/workflows/lock.yml

```
@@ -0,0 +1,19 @@
+name: 'Lock Threads'
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  action:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dessant/lock-threads@v2
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          issue-lock-inactive-days: '14'
```

We may want to leave a bit more time here -- If we close an issue when a fix is merged, it may take a week for the edge release to become available, and that only leaves a week to test the change. WDYT about 30 days?
```suggestion
          issue-lock-inactive-days: '30'
```